library/core/src/slice/iter.rs

```diff
@@ -2114,6 +2120,12 @@ unsafe impl<'a, T> TrustedRandomAccessNoCoerce for ChunksExactMut<'a, T> {
     const MAY_HAVE_SIDE_EFFECT: bool = false;
 }
 
+#[stable(feature = "chunks_exact", since = "1.31.0")]
+unsafe impl<T> Send for ChunksExactMut<'_, T> where T: Send {}
```

Comment:
Debatable nit, but pointing it out just for consideration:

```suggestion
unsafe impl<'r, T> Send for ChunksExactMut<'r, T> where &'r mut [T]: Send {}
```

(this makes it "easier" to understand the causality relation, imho)